dpc-api/src/test/java/gov/cms/dpc/api/resources/v1/AdminResourceTest.java

```diff
@@ -53,4 +38,26 @@ void testNoGoldenMacaroon() throws IOException, URISyntaxException {
         assertEquals(HttpStatus.UNAUTHORIZED_401, conn.getResponseCode());
         conn.disconnect();
     }
+
+    @Test
+    void testSearchByNpi() {
+        IGenericClient client = APIAuthHelpers.buildAdminClient(ctx, getAdminResourceURL(), GOLDEN_MACAROON, false);
+
+        Map<String, List<String>> searchParams = new HashMap<>();
+        searchParams.put("npis", Collections.singletonList(DPCIdentifierSystem.MBI.getSystem() + "|" + APITestHelpers.ORGANIZATION_NPI));
```

Comment:
This should be NPPES as the system, right?

```suggestion
        searchParams.put("npis", Collections.singletonList(DPCIdentifierSystem.NPPES.getSystem() + "|" + APITestHelpers.ORGANIZATION_NPI));
```

Reply:
Yep, you're right.  

Technically it doesn't matter much, though, because when the request eventually gets to dpc-attribution we strip the system off, anyway.  The request will fail as invalid if we don't have a system at all, but we can just put some random text there and it'll work, regardless.